### PR TITLE
Search my points

### PR DESCRIPTION
**Pull request by ARenzDev**
Issue: N/A

---

[N/A] Add command to get query for search my points worked on specific month

# Overview

I added the get query functionality to search for my points worked in a specific month, also TaskComplete, addpoint and addboostedpoint tags.

# How to test

1. /my-points-query
2. You are given 3 options, year, month and user. If you do not choose the year, it will put the current one, if you do not choose the month it will put the previous month by default and if you do not choose the user it will put yours by default.

### CODE REVIEW

File: src/commands/utility/search-my-points.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+const { SlashCommandBuilder } = require("discord.js");
+require("dotenv").config();
```
Comment (heliomar-pena) [inline src/commands/utility/search-my-points.js:2]: Esto mejor moverlo a un lugar centralizado, como aca: https://github.com/TeamNovaSoft/discord-bot/blob/main/src/index.js

File: src/commands/utility/search-my-points.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      option
+        .setName("channel")
+        .setDescription("Select a specific channel")
+        .setRequired(false)
+        .addChoices(
+          { name: "novabot", value: "novabot" },
+          { name: "code-review", value: "code-review" }
+        )
+    ),
+
```
Comment (heliomar-pena) [inline src/commands/utility/search-my-points.js:39]: if possible allow user select various

File: src/commands/utility/search-my-points.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    .addStringOption((option) =>
+      option
+        .setName("channel")
+        .setDescription("Select a specific channel")
+        .setRequired(false)
+        .addChoices(
+          { name: "novabot", value: "novabot" },
+          { name: "code-review", value: "code-review" }
+        )
```
Comment (heliomar-pena) [inline src/commands/utility/search-my-points.js:37]: move this to a .env to make it easy to change

Comment (ARenzDev) [reply src/commands/utility/search-my-points.js]: I decided to delete them from env and it is better to enter them manually

image
image

File: src/commands/utility/search-my-points.js (hunk 2 above)
Comment (ARenzDev) [reply src/commands/utility/search-my-points.js]: I did it this way
image

**Review by heliomar-pena — APPROVED** [review]

File: src/commands/utility/search-my-points.js (hunk 3 above)
Comment (heliomar-pena) [reply src/commands/utility/search-my-points.js]: Thanks!

File: src/commands/utility/search-my-points.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
+const { SlashCommandBuilder } = require("discord.js");
+require("dotenv").config();
```
Comment (heliomar-pena) [inline src/commands/utility/search-my-points.js:2]:
```suggestion
```

**Pull request merged**